src/java.desktop/unix/classes/sun/awt/X11/XInputMethod.java

```diff
@@ -45,25 +52,35 @@
 public class XInputMethod extends X11InputMethod {
     private static final PlatformLogger log = PlatformLogger.getLogger("sun.awt.X11.XInputMethod");
 
+    private InputMethodContext inputContext;
+
     public XInputMethod() throws AWTException {
         super();
     }
 
+    @Override
```

Comment:
why do these functions need the override tag now?

Reply:
cc @quantum6 

I tracked down this PR from your Zhihu answer:
https://www.zhihu.com/question/360671800/answer/3481640228

It's your irresponsiveness (towards this question) which lead this PR to be deprecated by the bot, and you blame the OpenJDK community for the atmosphere of laziness?

Reply:
cc @quantum6

I tracked down this PR from your Zhihu answer:
https://www.zhihu.com/question/360671800/answer/3481640228

It's your irresponsiveness (towards this question) which lead this PR to be deprecated by the bot, and you blame the OpenJDK community for the atmosphere of laziness?

But things might be more complicated than this. There might be other reviewers who have limited time / interests in looking at your PR (which is extremely frustrating).

Reply:
> cc @quantum6
> 
> I tracked down this PR from your Zhihu answer: https://www.zhihu.com/question/360671800/answer/3481640228
> 
> It's your irresponsiveness (towards this question) which lead this PR to be deprecated by the bot, and you blame the OpenJDK community for the atmosphere of laziness?
> 
> But things might be more complicated than this. There might be other reviewers who have limited time / interests in looking at your PR (which is extremely frustrating).

Thanks very much. This question is so common that many people should face it.
Here is a example to confirm it:
https://github.com/quantum6/X11InputMethod